README.md

```diff
@@ -217,7 +217,7 @@ Make sure to first change the `kubectl` context to the GKE cluster with `gcloud`
 
 app-config for mounting [appsettings.json](src/api/appsettings.json).
 
-- `kubectl create configmap app-config --from-file=src/api/appsettings.json`
+- `kubectl create configmap app-config --from-file=appsettings.json=./.kube/appsettings.kube.json`
```

Comment:
Creating the configmap this way allows us to mount other json files as any of the appsettings.*.json files

Reply:
oh ok